lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -18,14 +19,14 @@ export default function DashboardApp() {
           src="/static/images/hypothesis-wordmark-logo.png"
           className="h-10"
         />
-      </div>
-      <div className="flex-grow px-3">
-        <div className="mx-auto max-w-6xl">
+      </header>
+      <InitialLoadingTracker classes="flex-grow">
```

Comment:
We can wrap all components in the dashboard with this, and it will be used for all future pages we add.